src/main/java/org/reactome/release/qa/common/QACheckerHelper.java

```diff
@@ -54,14 +53,14 @@ public static boolean isChimeric(GKInstance rle) throws Exception {
 	
 	@SuppressWarnings("unchecked")
 	public static GKInstance getHuman(MySQLAdaptor dba) throws Exception {
-	    Collection<GKInstance> c = dba.fetchInstanceByAttribute(ReactomeJavaConstants.Species,
-	            ReactomeJavaConstants._displayName,
-	            "=", 
-	            "Homo sapiens");
-	    if (c == null || c.size() == 0)
-	        throw new IllegalStateException("Cannot find species Homo sapiens in the database, " + 
-	                                        dba.getDBName() + "@" + dba.getDBHost());
-	    return c.iterator().next();
+		Collection<GKInstance> c = dba.fetchInstanceByAttribute(ReactomeJavaConstants.Species,
+				ReactomeJavaConstants._displayName,
+				"=",
+				"Homo sapiens");
```

Comment:
Could the result of this query be cached somewhere so that it doesn't need to be executed every time the `getHuman` method is called?

Reply:
Actually this is a method that was created from Fred, I just reverted it after coming up with a solution that didn't require checking of a GKInstance method. That being said, I can implement that caching suggestion since the method is terribly inefficient

Reply:
@cookersjs Actually, it doesn't look like your code is even calling this method anymore?

Reply:
No its not. The story: This is a method that had already existed in release-qa/.../QACheckerHelper, I just hadn't realized it. During the earlier iterations of work today where the code still had places checking the Human instance, I decided to utilize this code instead of create new (and redundant), Human-instance-getting code. The only problem was that the method was really slow due to the `fetchSpeciesByAttribute` usage -- so I rewrote the method to search specifically for the instance with DbId 48887. This was quicker and got the runtime down to the usual 1-2 minutes. 

After the dust settled and we came up with a way to check for Homo sapiens without using the Homo sapiens instance, I figured it might as revert the code back to the state it was, since it wasn't mine and I wasn't using it anymore.

Reply:
Excellent story :)

Reply:
If this original code is still used by other QAChecks, I'd leave it as it is, but with a TODO note for future development, maybe saying "this could be much more efficient if the GKInstance were only fetched once and cached".

This PR is (mostly) for _your_ code, and if it's no longer necessary to change it, then don't change it.